src/engine/EngineUtils.h

```diff
@@ -53,19 +61,21 @@ void LoadGame(struct GameData* gameData,
 
     for (unsigned int i = 0; i < NumberOfSlots; ++i)
     {
-        BindVBO(CreateVBO());
+        assert(gameData->RenderSlotArray[i].Model.Name);
+        if(ModelNameSetVBO.find(gameData->RenderSlotArray[i].Model.Name) == std::end(ModelNameSetVBO))
+        {
+            ModelNameSetVBO.insert((gameData->RenderSlotArray[i].Model.Name));
+            BindVBO(CreateVBO());
 
-        // Only allocate what is needed
-         
-        //TODO handle this for dev/release build 
-        // Will crash if these are zero
-        assert(gameData->RenderSlotArray[i].Model.VBOMemoryAllocationSize_PosOnly);
-        assert(gameData->RenderSlotArray[i].Model.VBOMemoryAllocationSize);
+            // Only allocate what is needed
```

Comment:
This comment no longer makes sense (we are not doing the tex/pos optimisation). Remove.